docs/content/preview/manage/ysql-major-upgrade-local.md

```diff
@@ -0,0 +1,237 @@
+---
+title: YSQL major upgrade
+headerTitle: YSQL major upgrade
+linkTitle: YSQL major upgrade
+description: Upgrade YugabyteDB to PostgreSQL 15
+headcontent: Upgrade YugabyteDB to a version that supports PG 15
+menu:
+  preview:
+    identifier: ysql-major-upgrade-2
+    parent: manage-upgrade-deployment
+    weight: 706
+type: docs
+---
+
+Upgrading YugabyteDB from a version based on PostgreSQL 11 (all versions prior to v2.25) to a version based on PostgreSQL 15 (v2.25.1 or later) requires additional steps. For instructions on upgrades within a major PostgreSQL version, refer to [Upgrade YugabyteDB](../upgrade-deployment/).
+
```

Comment:
The upgrade process will be fully online, with full read and write access to all the data!

@ddhodge can you expand on that statement and make it sound awesome. 
This is a very unique differentiator that no other PG vendor offers today.